### ISSUE TITLE
Rollup plugin?

### ISSUE THREAD

**Issue by phr3nzy** [timhall/sapper-spa#3]
Just checking to see if there are plans of porting this to rollup. cheers

**Comment by pramod-thaz** [comment]
Yeah. Curious to know how to use this with rollup. Please advise.
